mesheryctl/pkg/utils/portforward.go

```diff
@@ -44,7 +44,7 @@ func NewPortForward(
 	ctx context.Context,
 	client *meshkitkube.Client,
 	namespace, deployName string,
-	host string, localPort, remotePort int,
+	host string, localPort, remotePort int, // I think "remotePort" should be idenitified dynamically based on the retrieved Pod configuration, instead of assuming 8080, or asking from user?
```

Comment:
Yes, although, for the service to be listening on a port other than 8080, the user will have to be running custom Meshery image builds.